Makefile

```diff
@@ -0,0 +1,7 @@
+build_r:
+	cp LICENSE clients/r-client
+	R CMD build clients/r-client
+
+build_py:
+	cp LICENSE clients/py-client
+	cd py-pkg && python3 setup.py sdist
```

Comment:
ok last thing...`py-pkg/` is not a directory in this project. Could you change this to `cd . clients/py-client`?

Reply:
Done- can't believe I missed that one!